### DISCUSSION TITLE
添加{subfigure}包报错

### DISCUSSION THREAD

**Discussion by ZJU-KMnO4** [TheNetAdmin/zjuthesis#278] [General]
添加{subfigure}包后，编译报错：Command \c@lofdepth already defined. \newcounter{lofdepth}. 网上的解决方法是注销lofdepth，注销后可以正常编译，而且实现了图片并排排列的效果。但是我不清楚注销这两行会带来什么影响，请问注销后对模板有什么影响？
![image](https://user-images.githubusercontent.com/74952844/201505298-1928b1c0-ea03-4d42-be66-39f778225293.png)

**Comment by TheNetAdmin** [comment]
我也不清楚，可能某些图片编号会错乱吧。尝试改用 `subcaption` 包提供的子图环境吧，应该效果好很多也不需要这样的hack。

**Comment by OctopussGarden** [comment]
`packages.tex`加下面几行解决包冲突
```latex
\makeatletter
\let\c@lofdepth\relax
\let\c@lotdepth\relax
\makeatother
\usepackage{subfigure}
```

**Comment by ZJU-KMnO4** [comment]
非常感谢。但是，我现在找不到自己注释的那两行在哪个文件了，不知道怎么改回去了。


-----原始邮件-----
发件人:OctopussGarden ***@***.***>
发送时间:2022-11-19 10:14:44 (星期六)
收件人: TheNetAdmin/zjuthesis ***@***.***>
抄送: ZJU-KMnO4 ***@***.***>, Author ***@***.***>
主题: Re: [TheNetAdmin/zjuthesis] 添加{subfigure}包报错 (Discussion #278)






packages.tex加下面几行解决包冲突

***@***.******@***.***\relax\makeatother\usepackage{subfigure}

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you authored the thread.Message ID: ***@***.***>






Qiang Ma， Ph.D. student




MOE Key Laboratory of Soft Soils and Geoenviromental EngineeringDepartment of Civil Engineering (Anzhong Building B-403)Zhejiang University (Zijingang Campus)866 Yuhangtang Road, Hangzhou 310058 PR. China





Tel: (+86) 18069870565




Email: ***@***.***

**Comment by kjfu** [comment]
packages.tex添加下面两行
`\usepackage{subfigure}`
`\usepackage[subfigure]{tocloft}`
